packages/@lwc/engine-server/README.md

```diff
@@ -1 +1,115 @@
 # @lwc/engine-server
+
+This package can be used to render LWC components as strings in a server environment.
+
+Usage of internal APIs are prevented by the compiler and are therefore not documented here.
+
+## Supported APIs
+
+This package supports the following APIs.
+
+### renderComponent()
+
+This function renders a string-representation of a serialized component tree, given a tag name
+and an LWC constructor. The output format itself is aligned with the [current leading
+proposal][explainer], but is subject to change.
+
+### LightningElement
```

Comment:
Same observation about moving the common APIs documentation to the `@lwc/engine-core` package.